apps/mobile-alphabet/src/app/styles.ts

```diff
@@ -0,0 +1,65 @@
+import { StyleSheet } from 'react-native';
+
+export const colors = {
+    primary: '#5f43b2',
+    secondary: ' #3a3153',
+    text: '#ffff',
+    accent: '#3a3153',
+    button: ' #FF9800',
+};
+
+const styles = StyleSheet.create({
+    background: {
+        backgroundColor: 'white',
+        height: 1000,
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+});
+
+export const detailStyles = StyleSheet.create({
+    background: {
+        backgroundColor: 'white',
+        height: 1000,
+        flex: 1,
+        justifyContent: 'center',
+    },
+});
+
+export const alphabetDetailStyle = StyleSheet.create({
```

Comment:
This is good, but it all needs to be configurable. You should inject a config from which you read the currently hard-wired values.